src/main/resources/nl/knaw/dans/catalog/resource/view/error.ftl

```diff
@@ -16,15 +16,15 @@
                     <p><em>${errorMessage.message}</em></p>
                     <p>There was an error displaying this page.</p>
                 </header>
+
                 <#if errorMessage.details?has_content>
-                <div class="bg-light my-5 py-4 px-4 border">
-                    <pre class="my-0"><code>${errorMessage.details}</code></pre>
-                </div>
-               </#if>
+                    <div class="bg-light my-5 py-4 px-4 border">
```

Comment:
ftl is new to me

Reply:
It is a freemarker template, which allows for text to be interpolated with values. In this case it is a html document.

Reply:
See https://freemarker.apache.org/ for more info